python_ta/reporters/templates/template.html

```diff
@@ -0,0 +1,100 @@
+<html>
+   <head>
+      <style type="text/css">
+         {% include "stylesheet.css" %}
+      </style>
+   </head>
+   <div id="preheader">
+      <div id="module">
+         Module analysed: {{ mod_name }}
+      </div>
+      <span id="date">
+      {{ date_time }}
+      </span>
+   </div>
+   <div id="wrap">
+      <div class="title">
+         PyTA Error Report
+      </div>
+   </div>
+   <div id="inside">
+      {% for file in messages_by_file %}
+      <div id="core">
+         <h3> {{ file.filename }} </h3>
+         <div id="code">
+            {{ code_err_title }}
+         </div>
+         {% for message in file.code %}
+         <div class="slider">
+            <p style="font-size: 110%">
```

Comment:
Use the css document, rather than inline styles. This occurs in several other places, which also needs changing.